lib/rest/publizon/local-adapter/mutator/fetcher.ts

```diff
@@ -34,6 +34,8 @@ export const fetcher = async <ResponseType>({
 
     if (!response.ok) {
       console.error(response.status, response.statusText, serviceUrl)
+      const data = await response.json()
+      throw Error(JSON.stringify(data))
```

Comment:
Would be nice to move all code that are shared between the two publizon fetchers into one place. So we are sure that we have the same behaviour.